prime-dotnet-webapi/Services/DefaultEnrolleeService.cs

```diff
@@ -426,20 +434,21 @@ public bool IsStatusChangeAllowed(Status startingStatus, Status endingStatus)
             return availableStatuses.Contains(endingStatus);
         }
 
-        public async Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, short statusCodeToCheck)
+        public async Task<bool> IsEnrolleeInStatusAsync(int enrolleeId, params short[] statusCodesToCheck)
         {
             var enrollee = await _context.Enrollees
                 .AsNoTracking()
                 .Include(e => e.EnrolmentStatuses)
                 .SingleOrDefaultAsync(e => e.Id == enrolleeId);
+
             if (enrollee == null)
             {
                 return false;
             }
 
             var currentStatusCode = enrollee.CurrentStatus?.StatusCode;
 
-            return statusCodeToCheck.Equals(currentStatusCode);
+            return statusCodesToCheck.Contains(currentStatusCode.Value);
```

Comment:
if currentStatusCode is null, calling currentStatusCode.Value will throw an error. Its probably better to do an explicit null check and return false or do something like `var currentStatusCode = enrollee.CurrentStatus?.StatusCode ?? -1;`

Reply:
Nice catch, definitely.  I like your suggestion I think I might have done a null check out of the box, but this is cleaner